src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -42,6 +42,7 @@ namespace
 static constexpr size_t column_page_size = AIE_COLUMN_PAGE_SIZE;
 static constexpr uint8_t Elf_Amd_Aie2p  = 69;
 static constexpr uint8_t Elf_Amd_Aie2ps = 64;
+static constexpr uint8_t Elf_Amd_npu3   = 65;
```

Comment:
There is no good rationale to add a new os_abi since both aie2ps and npu3 are using 57 bits address internally. But we might consider renaming Elf_Amd_Aie2ps so it can include npu3